qa/tasks/quiescer.py

```diff
@@ -186,14 +186,15 @@ def dump_ops_all_ranks(self, dump_tag):
 
             self.logger.debug(f"Dumping ops on rank {rank} ({name}) to a remote file {remote_path}")
             try:
-                _ = self.fs.rank_tell(['ops', '--flags=locks', f'--path={daemon_path}'], rank=rank)
-                remote_dumps.append((info, remote_path))
+                p = self.fs.rank_tell(['ops', '--flags=locks', f'--path={daemon_path}'], rank=rank, wait=False)
```

Comment:
My mistake, I cannot do this:

https://pulpito.ceph.com/pdonnell-2024-05-08_22:06:20-fs-wip-pdonnell-testing-20240508.183908-debug-distro-default-smithi/7698868/

```
2024-05-09T00:35:43.460 ERROR:tasks.quiescer.fs.[cephfs]:Couldn't pull ops dump at '/var/run/ceph/b96c13bc-0d98-11ef-bc97-c7b262605968/ops-7749c26b-1-mds.i.json' on rank 2 (i), error: 'dict' object has no attribute 'wait'
```

Reply:
:'( I was looking forward to the PR... How hard is it to add the async capability to the `rank_tell`?

Reply:
Instead of using the helper `rank_tell`, probably just manually submit the command instead.